chat.js

```diff
@@ -12,35 +13,95 @@ const chatLog = [];
 let first_person = '';
 let second_person = '';
 let current_person = '';
+let fileChoosen = '';
 
-rl.question('Enter first person name: ', (name1) => {
-    first_person = name1;
-    rl.question('Enter second person name: ', (name2) => {
-        second_person = name2;
-        current_person = first_person;
-        chat();
-    });
+rl.question('Continue from previous chat? y/n :', (cont) => {
+	if (cont === "y") {
+		
+		const files = fs.readdirSync(path.join(__dirname)).filter((filename) => filename.includes(".md") && !filename.includes("README"));
+
+		if (files.length === 0) {console.log("No Files Available")}
+		
+		let iterator = 0;	
+		files.forEach((File1) => {
+			console.log(File1, iterator);
+			iterator++;
+		})
+		
+		rl.question("Please choose a file, choose by the number next to the file: ", (itera) => {
+			fileChoosen = files[itera];
+			fs.readFile(fileChoosen, 'utf-8',(err, data) => {
+				data = data.split("\n");
+				
+				if (err) {
+					throw err;	
+				}
+
+				try {
+					first_person = data[0].split(":")[0].replaceAll("*", "");
+					second_person = data[1].split(":")[0].replaceAll("*", "");
+					current_person = first_person;
+					chatCont();
+				} catch {
+					console.log("Log unfinished, or log is not accessible. The log either has only 1 persons input or it has been most likely corrupted.")
+					rl.close();
+				}
+			
+			});
+			
+		});
+
+	} else {
+		rl.question('Enter first person name: ', (name1) => {
+			first_person = name1;
+			rl.question('Enter second person name: ', (name2) => {
+				second_person = name2;
+				current_person = first_person;
+				chat();
+			});
+		});
+
+	}
 });
 
+function chatCont() {
```

Comment:
**issue:** looks like we're missing a line break when we merge updates to an existing chat eventually

<img width="573" alt="Screenshot 2024-03-15 at 10 27 16" src="https://github.com/fyodorio/chat-with-yourself/assets/19417873/d7705888-4398-434b-9759-7d532154350a">

Reply:
I will figure this out, and the soultion to it, and get back to you. Thank you for finding the error, and for working with me on this.

Reply:
Hello @fyodorio

![Screenshot from 2024-03-15 09-19-07](https://github.com/fyodorio/chat-with-yourself/assets/138682950/5acf20d6-ab3a-492c-989c-fd7bb899e59a)

For some reason, the line breaks work fine for me. But, a possible solution to the problem is to initially push a new line into chatLog when we type Y for continue

`chatLog.push("  \n");`

This is a change I have implemented. I hope it works, thank you!!